### PR TITLE
Store settings ms2deepscore

### PR DESCRIPTION
**Pull request by niekdejonge**
Store the settings for training of an MS2Deepscore model as a json file

### CODE REVIEW

File: tests/test_training_wrapper_function.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    train_ms2deepscore_wrapper
+from tests.create_test_spectra import pesticides_test_spectra
+
+
+def load_pickled_file(filename: str):
+    with open(filename, 'rb') as file:
+        loaded_object = pickle.load(file)
+    return loaded_object
+
+
```
Comment (florian-huber) [inline tests/test_training_wrapper_function.py:17]:
```suggestion
```

Comment (florian-huber) [reply tests/test_training_wrapper_function.py:17]: Seems unnecessary.

**Review by florian-huber — APPROVED** [review]
Nice job @niekdejonge !
Good additions and refactoring. I made a few tiny changes (renaming of a function and removing some unnecessary pieces) which are hopefully OK. I'll merge it right away.

**Comment by niekdejonge** [comment]
Thank  you for checking and merging @florian-huber!

**Pull request merged**